dist/index.html

```diff
@@ -65,7 +65,6 @@ <h2 class="separator" aria-hidden="true"> | </h2>
         </section>
       </section>
```

Comment:
html is really semantic good job on that !

Reply:
Thank you!